css/browser.css

```
@@ -416,3 +416,18 @@ html.private-mode ._1htf > span:first-child {
 	visibility: visible;
 }
 /* -- BLOCK END: Private Mode -- */
+
+/* -- BLOCK START: Small conversations window -- */
+
+@media all and (max-width: 500px) {
```

Hooray for Media Queries Level 4 😉

```suggestion
@media (max-width: 500px) {
```